yarn-project/end-to-end/src/utils.ts

```diff
@@ -237,15 +237,15 @@ export function delay(ms: number): Promise<void> {
  */
 export async function calculateStorageSlot(slot: bigint, key: Fr): Promise<Fr> {
   const wasm = await CircuitsWasm.get();
-  const balancesStorageSlot = new Fr(slot); // this value is manually set in the Noir contract
-  const mappingStorageSlot = new Fr(4n); // The pedersen domain separator for storage slot calculations.
+  const mappingStorageSlot = new Fr(slot); // this value is manually set in the Noir contract
+  const mappingStorageSlotSeparator = new Fr(4n); // The pedersen domain separator for storage slot calculations.
 
   // Based on `at` function in
   // aztec3-packages/yarn-project/noir-contracts/src/contracts/noir-aztec3/src/state_vars/storage_map.nr
   const storageSlot = Fr.fromBuffer(
-    pedersenPlookupCommitInputs(
+    pedersenCompressInputs(
```

Comment:
Having an issue with pedersens not matching up when using the plookup variant here? any idea what im doing wrong here?